docs/Configuration.md

```diff
@@ -64,14 +64,21 @@ Default value: `0.5`.
 
 ## MYSQL section
 
-Available keys : `host`, `user`, `password`, `database`.
+Available keys : `host`, `port`, `user`, `password`, `database`.
 
 ### host
 
 The host name of the machine on which the MySQL server is running.
 
 If this property is unspecified, the value of [DB.database_host] is used instead.
 
+### port
+
+The port the MySQL server is listening on. If the server is on the local
+machine, then the [MYSQL.host] value must contain the loopback IP
+address (127.0.0.1).
+Default value: `3306`.
```

Comment:
If MYSQL.host = localhost, then MYSQL.port is not being used because the driver connects using a UNIX socket. I think this should be clear from the documentation.

Maybe we should also do something in the implementation in this case? Like making parse() fail or emit a warning. I'm leaning towards having parse() fail, but I'm not sure.

I guess the special `localhost` value should be documented in MYSQL.host as well.

Reply:
I've added some documentation, I'll add some code to catch this behavior. I think I'll go with a warning as this is not an error per se.